addon/styles/addon.scss

```diff
@@ -1,35 +1,38 @@
 @import 'node_modules/ember-frost-core/addon/styles/frost-theme';
+$infobarHeight: 80px;
+$infobarBorder: 1px;
 
 .frost-info-bar {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  height: 90px;
-
-  border-bottom: 1px solid $frost-color-lgrey-1;
-
+  height: $infobarHeight + $infobarBorder;
+  box-sizing: border-box;
+  border-bottom: $infobarBorder solid $frost-color-lgrey-1;
   background-color: $frost-color-white;
 
   .title {
     margin-left: 30px;
+    line-height: 1;
 
     .primary-title {
-      margin: 25px 0 10px 0;
-      font-size: $frost-font-xl;
-      color: $frost-color-night-1;
+      margin-top: 20px;
+      margin-bottom: 8px;
```

Comment:
Would it make sense to put these margin values in SASS variables at the top of this file?